### ISSUE TITLE
Template literals inside  expressions (`{}`) cause broken syntax highlighting

### ISSUE THREAD

**Issue by adamjohnson** [Destiner/sublime-astro#4]
## Screenshot showing a comparison:

![Screenshot comparing template literals in .astro files in ST4 and VS Code](https://user-images.githubusercontent.com/575865/149703072-a30e973e-d585-49ef-9f45-8610fdc4d8f0.png)

## Sample code

Copy and paste this to test:

```astro
---
const { href } = Astro.props;
const varOne = 'hello';
const varTwo = 'world';
---

<a class={`${varOne} ${varTwo} my-class is-here`} href={href} id="my-id">
  My Link
</a>

<div class="example">
  <ul class="my-list">
    <li>Item 1</li>
    <li>Item 2</li>
    <li>Item 3</li>
    <li>Item 4</li>
    <li>Item 5</li>
  </ul>
</div>

```

This might be related to #3.

**Comment by estudiooka** [comment]
+1

**Comment by deathaxe** [comment]
Fixed by Astro 2.0.0 (now on https://github.com/SublimeText/Astro)

**Comment by adamjohnson** [comment]
Works great, thanks!
